packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -118,30 +118,41 @@ export class AccountForm extends PureComponent {
     if (locales && lang) {
       extend(locales[lang])
     }
+
+    if (props.oauth) return
```

Comment:
I assume that an error is thrown if we don't make an early return here.

Maybe we should prevent error from `sanitizeFields` instead ?

Reply:
In the previous version there was a redirection to OAuth before sanitize and that's true, fields aren't used and there is no fields for OAuth version ...
So I keep the same idea in first step.